node/_crypto/randomFill.ts

```diff
@@ -0,0 +1,61 @@
+import randomBytes from "./randomBytes.ts";
+import { Buffer } from "../buffer.ts";
+
+export default function randomFill(
+  buf: Buffer,
+  cb: (err: Error | null, buf?: Buffer) => void,
+): void;
+export default function randomFill(
+  buf: Buffer,
+  offset: number,
+  cb: (err: Error | null, buf?: Buffer) => void,
+): void;
+export default function randomFill(
+  buf: Buffer,
+  offset: number,
+  size: number,
+  cb: (err: Error | null, buf?: Buffer) => void,
+): void;
+
+export default function randomFill(
+  buf: Buffer,
+  offset?: number | ((err: Error | null, buf?: Buffer) => void),
+  size?: number | ((err: Error | null, buf?: Buffer) => void),
+  cb?: (err: Error | null, buf?: Buffer) => void,
+) {
+  const callback = cb as (err: Error | null, buf?: Buffer) => void;
```

Comment:
This doesn't seem supporting the 1st and 2nd overloads

If I run the code like the below
```ts
import randomFill from "./node/_crypto/randomFill.ts";                   
const u8 = new Uint8Array(10);  
randomFill(u8, () => {  
  console.log(u8);  
});
```

It throws with `error: Uncaught TypeError: callback is not a function`

I think you need to check values and types of `offset`, `size`, and `cb` and swap variables

Reply:
Thanks! Working on that.

Reply:
Could you check again please @kt3k ?